test/shared/policies/serializationPolicyTests.ts

```diff
@@ -144,4 +147,91 @@ describe("serializationPolicy", () => {
       "dateTimeRfc1123Property": new Date("2011-10-05T14:48:00.000Z")
     });
   });
-});
\ No newline at end of file
+
+  it("[de]serializes XML request and response bodies", async () => {
+    const policyFactory: RequestPolicyFactory = serializationPolicy({ outputType: SerializationOutputType.XML });
+
+    const inMemoryEchoServer: RequestPolicy = {
+      send: async (request: HttpRequest) => {
+        return new InMemoryHttpResponse(request, 200, request.headers, request.serializedBody as string);
+      }
+    };
+
+    const policy: RequestPolicy = policyFactory(inMemoryEchoServer, new RequestPolicyOptions());
+
+    const bodySpec = compositeSpec({
+      typeName: "Root",
+      xmlRootName: "my-root",
+      propertySpecs: {
+        "foo": {
+          xmlName: "foo",
```

Comment:
Since the xmlName and the property name are the same (`foo`), xmlName shouldn't be necessary here.

Reply:
Perhaps needs to be expanded into coverage that handles a default case (xmlName === propertyName) as well as one where the xmlName is different than the property name.

Reply:
Sure. Another test where the xmlName isn't the same as the property name sounds good.